### DISCUSSION TITLE
CURRENT STATUS

### DISCUSSION THREAD

**Discussion by enrique-lovera** [nodejs-mobile/nodejs-mobile#13] [General]
Hello everyone! hope you are doing great today!! 

Do somebody know if this is still being supported/ developed ? we are thinking to implement it for a new project thanks in advance.

**Comment by staltz** [comment]
Yes, it's absolutely being maintained. I just updated Node.js 12 to Node.js 16 inside nodejs-mobile.

**Reply by enrique-lovera** [reply]
Staltz!! many thanks for your response! and thank you so much for the work you have done here! i will start implementing the node core lib and let you know how it goes :)
